.travis.yml

```diff
@@ -0,0 +1,11 @@
+env:
+  global:
+    secure: gkDTNI2TORFDqE8MgF8TnRGBaAP8IeJiWQr6O/gC4JfhzxZrwGlAefh6KRdd21YeJRjIgLJCxLd9rFLsCTiOK9PgAMic2IycnbJIueXfLgJpixhbveXAUhovxhZnAakSGaC6grbAMJkSTs3KtIe5zkDEOiPTxoDl3BnjSyLyK536UghtsRr2eaJpeFT7wiDfssPA39DDk9G5wC+xn3YadN+/zRw1lWzegJCCAJHDGGf85LvS9MxhvXa8safEpEEG1T6DZB7oc0c33J0tiAs5QKajlawm3yI+NFl4C4uBlasW4aEUewTuQEKqd5vy27JM3mcN7qO3uEhV4oPL29iVaC8iIRJ1exItoEgzoQQROqjHu93WILnpwhLqXPMmbickZLCngmtgwbSoRlaQ5ov+3ZGiz/7fTGHAWA3tQv39Nv99DGbYsMIYea9f4SqnjJjcO+dwYkT3jr2Vylb5cNV2O64SMt4BPU29z0btB+qrSMM9Ki7LbO2tv/0yUzmFdHTk1oDKF+7NRrRG7HFeFkAGzqHVD8Fd1OMiHsBrRr5itBlfi5olYEI+MEyhPkWY7lK94F26AMFV67dg4B+OaxPSPAUsp58sUp8Swi5oWF17Cs/23gagR16h07ozkeAhIw5+kGmWbAP+5Hp4H/FiVdkC/eYXShCuTC4qNrxfFEmjx6Y=
+branches:
+  only:
+    - master
```

Comment:
This might be a good time to change this repo to use `main` instead of `master`.

Reply:
Updated to use main and re-targeted PR